backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -198,6 +199,25 @@ public class CsvValidatorUtils {
   private static final Set<String> TEST_RESULT_STATUS_VALUES = Set.of("f", "c");
   public static final String ITEM_SCOPE = "item";
 
+  //  http://hl7.org/fhir/R4/codesystem-data-absent-reason.html#data-absent-reason-unknown
+  private static final Set<String> DATA_ABSENT_REASONS =
+      Set.of(
```

Comment:
do we need to include all of these? If the data absent reason is only for the patient's name I'm wondering if we should only include the ones that actually make sense for this and the ones we will support for the csv upload? 🤔

Reply:
I would want to include all the values that are in the spec on the backend and then configure our frontend / the GraphQL layer to apply process-specific validation. That way we're not getting into spec drift for anyone that might be using this constant elsewhere for other purposes. This way also if this field is getting used in ways that we don't anticipate that are nonetheless valid in the spec, we don't have to go back and add new fields.